AUTHORS.md

```diff
@@ -0,0 +1,30 @@
+<!--
+  - SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+# Authors
+
+- Alexander Weidinger <alexwegoo@gmail.com>
+- Bernhard Scheirle <bernhard+git@scheirle.de>
+- Boris Fritscher <boris.fritscher@gmail.com>
+- Christian Nöding <christian@noeding-online.de>
+- Christoph Wurst <christoph@winzerhof-wurst.at>
+- Clement Wong <mail@clement.hk>
+- Daniel Kesselberg <mail@danielkesselberg.de>
+- gouglhupf <dr.gouglhupf@gmail.com>
+- Hamza Mahjoubi <hamzamahjoubi22@proton.me>
```

Comment:
This is a typo tho  that I just noticed, should be hamzamahjoubi221@proton.me but it can be updated in the next batch